src/main/java/kr/codesquad/Main.java

```diff
@@ -1,7 +1,60 @@
 package kr.codesquad;
 
+import java.util.Scanner;
+
 public class Main {
+    static String[][] resultLine;
+    static Scanner sc = new Scanner(System.in);
+
+    // TODO : naming convention - 완료, depth 1로 줄이기 - 완료, 메서드는 10줄 이내 - 완료, 계산하는 로직 메서드로 빼기 - 완료
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        printMessageHuman();
+        int human = Integer.parseInt(input()); // 3
```

Comment:
변수명은 보다 의미를 잘 나타내도록 쓰고 리턴 타입도 적절하게 활용해 주세요.
```suggestion
        int numParticipants = inputNumParticipants();
```